packages/manager/apps/pci-block-storage/src/components/exten-banner-beta/ExtenBannerBeta.tsx

```diff
@@ -0,0 +1,62 @@
+import { useFeatureAvailability } from '@ovh-ux/manager-react-components';
+import { OsdsMessage, OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_MESSAGE_TYPE,
+  ODS_TEXT_COLOR_HUE,
+  ODS_TEXT_COLOR_INTENT,
+} from '@ovhcloud/ods-components';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import { FA_EXTEN_BANNER } from '@/api/data/quota';
+import { useProjectsAvailableVolumes } from '@/api/hooks/useProjectsAvailableVolumes';
+
+const extenProducts = [
+  'volume.high-speed-BETA.consumption',
+  'volume.classic-BETA.consumption',
+];
+
+function Banner() {
+  const { t } = useTranslation('exten-banner-beta');
+  const { projectId } = useParams();
+  const { data: availableVolumes } = useProjectsAvailableVolumes(projectId);
+
+  const regions = useMemo(
+    () =>
+      availableVolumes
+        ? [
+            ...new Set(
+              availableVolumes.plans
+                .filter((p) => extenProducts.includes(p.code))
+                .flatMap((p) => p.regions)
+                .map((r) => r.name),
```

Comment:
In addition to my below comment, I'll suggest to use `reduce` to return a `string`.

```ts
availableVolumes.plans.reduce(...)
```